Modules/_elementtree.c

```diff
@@ -813,12 +814,16 @@ _elementtree_Element___deepcopy___impl(ElementObject *self, PyObject *memo)
 
     PyTypeObject *tp = Py_TYPE(self);
     elementtreestate *st = get_elementtree_state_by_type(tp);
-    tag = deepcopy(st, self->tag, memo);
+    tmp = Py_NewRef(self->tag);
+    tag = deepcopy(st, tmp, memo);
+    Py_DECREF(tmp);
```

Comment:
```suggestion
    Py_CLEAR(tmp);
```
This is slightly safer because it NULLs out the variable so you can't accidentally use it later.

Reply:
Yes, let's be on the safe side, thanks!